CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [ENHANCEMENT] Distributor: Added distributors ring status section in the admin page. #4151
 * [ENHANCEMENT] Added zone-awareness support to alertmanager for use when sharding is enabled. When zone-awareness is enabled, alerts will be replicated across availability zones. #4204
 * [ENHANCEMENT] Added `tenant_ids` tag to tracing spans #4147
+* [ENHANCEMENT] Made handler function for `/config` endpoint overrideable so it can be extended by downstream projects.
```

Comment:
Don't forget the PR number (yes, it is a chicken/egg situation)

```suggestion
* [ENHANCEMENT] Made handler function for `/config` endpoint overrideable so it can be extended by downstream projects. #4225
```